OIIO master build failed with Visual Studtio 2013

The compiler gives the following errors:

Error   360 error C4430: missing type specifier - int assumed. Note: C++ does not support default-int   C:\oiio\src\include\OpenImageIO\fmath.h 406 1   OpenImageIO
Error   361 error C4430: missing type specifier - int assumed. Note: C++ does not support default-int   C:\oiio\src\include\OpenImageIO\fmath.h 412 1   OpenImageIO
Error   451 error C2039: 'min' : is not a member of 'std'  
C:\oiio\src\include\OpenImageIO\simd.h  1082    1   simd_test
Error   456 error C2039: 'max' : is not a member of 'std'  
C:\oiio\src\include\OpenImageIO\simd.h  1095    1   simd_test

I added "#include <algorithm>" in "simd.h" to remove the last two errors, but not sure how to fix the first two.

--- a/src/include/OpenImageIO/fmath.h
+++ b/src/include/OpenImageIO/fmath.h
@@ -402,12 +402,12 @@ template<typename T> struct is_same<T,T> { static const bool value = true; };
 
 
 
-template <typename IN, typename OUT>
-inline OUT bit_cast (const IN in) {
+template <typename IN_TYPE, typename OUT_TYPE>
+inline OUT_TYPE bit_cast (const IN_TYPE in) {
     // NOTE: this is the only standards compliant way of doing this type of casting,
     // luckily the compilers we care about know how to optimize away this idiom.
-    OUT out;
-    memcpy (&out, &in, sizeof(IN));
+    OUT_TYPE out;
+    memcpy (&out, &in, sizeof(IN_TYPE));
     return out;
 }
 

